setup.py

```diff
@@ -9,6 +9,12 @@
     print("Please upgrade to Python 3.8 or higher.")
     sys.exit(1)
 
+# # sasl does not support python 3.11 and higher
```

Comment:
This looks like an outdated comment. What is it referring to?

Reply:
removed!